docs/openapi-docs/src/main/scala/tapir/docs/openapi/schema/TSchemaToOSchema.scala

```diff
@@ -8,10 +9,10 @@ import tapir.{Schema => TSchema}
   * Converts a tapir schema to an OpenAPI schema, using the given map to resolve references.
   */
 private[schema] class TSchemaToOSchema(schemaReferenceMapper: SchemaReferenceMapper, discriminatorToOpenApi: DiscriminatorToOpenApi) {
-  def apply(schema: TSchema): ReferenceOr[OSchema] = {
+  def apply(schema: (TSchema,Validator[_])): ReferenceOr[OSchema] = {
     schema match {
-      case TSchema.SInteger =>
-        Right(OSchema(SchemaType.Integer))
+      case (TSchema.SInteger, ValueValidator(constraints)) =>
+        Right(OSchema(SchemaType.Integer).copy(minimum = constraints.collectFirst { case Constraint.Minimum(v) => v }))
```

Comment:
This pattern feels like it might be tough to scale as the number of constraints grows. Would it be better to add an `OSchema => OSchema` method onto `Constraint` so the custom logic for each constraint is located in a single place?